### ISSUE TITLE
[Bug] Can't exit reader if chapter fails to load

### ISSUE THREAD

**Issue by TristanWasTaken** [manga-download/hakuneko#6218]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
yes

**Describe the bug**
When you open a chapter in the reader and you click on the next or previous chapter button and the chapter somehow fails to load (it maybe helps that I have a weird issue I'm currently trying to debug where the dev tools network tab doesn't show a single request and hakuneko timeouts the download after some time), the reader options won't show up.
One of the reader options is the reader exit button, which means you have to either 1. wait an arbitrary amount of time until it appears or 2. restart hakuneko.
I'm not sure if it has to be a timeout error. If it really is connected to my timeout issue, then my guess is that it only loads the reader options when it got a callback from some handler (probably because of the chapter number/title and next/previous chapter button).

**To Reproduce**
1. Go to a manga with at least two chapters
2. Click on "show preview of chapter's pages" button
3. click on the next chapter button
4. Have the chapter fail to load (In my case it's the weird timeout issue)
5. Reader options are missing

**Expected behavior**
Reader options always appear

**Screenshots**
Missing reader options
<img width="1917" alt="Screenshot 2023-09-11 110303" src="https://github.com/manga-download/hakuneko/assets/43283797/cf104230-732c-4f5b-9b8e-d0be59bba072">

Reader options appear after an arbitrary amount of time (the download timeout was probably triggered?)
<img width="1916" alt="Screenshot 2023-09-11 111534" src="https://github.com/manga-download/hakuneko/assets/43283797/836bb776-e87a-4c72-a9fe-911e79240fe4">

**Context (please complete the following information):**
 - OS: Windows 11
 - HakuNeko Version: 6.17@[513508](https://github.com/manga-download/hakuneko/commit/513508ce25a89f065211ffcdf435630b47465585)

**Additional context**
Discord thread on my timeout issue: [link](https://discord.com/channels/581629183159959582/1139829826958659684/1139829826958659684)

**Comment by Sheepux** [comment]
Yep, i know this issue. Will not be solved in hakuneko. However such issue does not happen within haruneko. I've handled the viewer to be fully asynchronous.
